posts/frangeris/03.secure-values-by-private-keys-in-aws.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Ensuring secure values by private keys in AWS (KMS, SSM, Secrets Manager)
+published: false
+description: Secure way of store and access sensible values on AWS
+tags: "aws, private keys security, serverless security, cloud security"
+cover_image: ./assets/03.cover.png
+---
+
+## Ensuring secure values by private keys in AWS (KMS, SSM, Secrets Manager)
```

Comment:
I'd use either "...with private keys..." or "...by using private keys..."

Reply:
with / by using sound a little bit redundant to me

Reply:
I just think `by` should be followed by a verb

Reply:
Not always, in this case is a preposition, but I got the idea, thanks anyways for the suggestion